Gemfile.lock

```diff
@@ -60,22 +60,29 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
+    addressable (2.8.7)
+      public_suffix (>= 2.0.2, < 7.0)
     asciidoctor (2.0.16)
     ast (2.4.2)
     awesome_print (1.9.2)
     better_errors (2.9.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
+    bigdecimal (3.1.8)
     binding_of_caller (1.0.0)
       debug_inspector (>= 0.0.1)
     builder (3.2.4)
     byebug (11.1.3)
     coderay (1.1.3)
+<<<<<< main
+    concurrent-ruby (1.3.4)
+    crack (1.0.0)
+      bigdecimal
+======
     concurrent-ruby (1.2.3)
     crack (0.4.5)
+>>>>>> main
```

Comment:
Umm. This is a merge conflict, one of 8 conflicts, to be precise. These are supposed to be resolved before committing.